src/components/productrecommendations.main.jsx

```diff
@@ -99,58 +101,62 @@ class ProductRecommendationsDisplayMain extends React.Component {
+              <div className="container">
+                <div className="carousel-inner">
+                  {this.renderProducts(product, length, 2)}
+                </div>
               </div>
+              <a className="left carousel-control" href="#theCarousel" data-slide="prev">
+                <i className="glyphicon glyphicon-chevron-left" />
+              </a>
+              <a className="right carousel-control" href="#theCarousel" data-slide="next">
+                <i className="glyphicon glyphicon-chevron-right" />
+              </a>
             </div>
-            <a className="left carousel-control" href="#theCarousel" data-slide="prev">
-              <i className="glyphicon glyphicon-chevron-left" />
-            </a>
-            <a className="right carousel-control" href="#theCarousel" data-slide="next">
-              <i className="glyphicon glyphicon-chevron-right" />
-            </a>
           </div>
-        </div>
-      </div>);
+        </div>,
+      );
     }
 
-    if (this.state.productData._recommendations[0]._replacement[0].links.length > 0) {
-      var product = this.state.productData._recommendations[0]._replacement[0].links;
-      var length = this.state.productData._recommendations[0]._replacement[0].links.length;
-      console.log(this.state.productData._recommendations[0]._replacement[0]);
-      data.push(<div key={`_${Math.random().toString(36).substr(2, 9)}`}>
-        <label className="control-label">
-          Product Replacements
-        </label>
-        <div className="col-md-12">
-          <div className="carousel slide" data-ride="carousel" id="the_replacementCarousel">
-            <div className="container">
-              <div className="carousel-inner">
-                {this.renderProducts(product, length, 2)}
+    if (productData._recommendations[0]._replacement[0].links.length > 0) {
+      const product = productData._recommendations[0]._replacement[0].links;
+      const { length } = product.length;
+      console.log(productData._recommendations[0]._replacement[0]); // eslint-disable-line no-console
+      data.push(
+        <div key={`_${Math.random().toString(36).substr(2, 9)}`}>
+          <label className="control-label" htmlFor="carousel2">
+            Product Replacements
+          </label>
+          <div className="col-md-12">
+            <div className="carousel slide" data-ride="carousel" id="the_replacementCarousel">
+              <div className="container">
+                <div className="carousel-inner">
+                  {this.renderProducts(product, length, 2)}
+                </div>
               </div>
+              <a className="left carousel-control" href="#the_replacementCarousel" data-slide="prev">
+                <i className="glyphicon glyphicon-chevron-left" />
+              </a>
+              <a className="right carousel-control" href="#the_replacementCarousel" data-slide="next">
```

Comment:
Why do you need to declare this as an href? If you're not navigating anywhere, you could change this element to a div

Reply:
standard recommendation from bootstrap to use hrefs